examples/animation/src/lib.rs

```diff
@@ -1,38 +1,37 @@
 use std::{path::Path, sync::Arc};
+use rend3::types::DirectionalLightHandle;
 
-const SAMPLE_COUNT: rend3::types::SampleCount = rend3::types::SampleCount::One;
+const SAMPLE_COUNT: rend3::types::SampleCount = rend3::types::SampleCount::Four;
 
 /// The application data, can only be obtained at `setup` time, so it's under an
 /// Option in the main struct.
 struct InitializedData {
     loaded_scene: rend3_gltf::LoadedGltfScene,
     loaded_instance: rend3_gltf::GltfSceneInstance,
     animation_data: rend3_anim::AnimationData,
-    _directional_light_handle: rend3::types::DirectionalLightHandle,
     animation_time: f32,
     last_frame_time: instant::Instant,
 }
 
 #[derive(Default)]
 struct AnimationExample {
     /// The application data, or `None` if it hasn't been initialized already
-    data: Option<InitializedData>,
+    data: Vec<InitializedData>,
```

Comment:
Should update the comment above (the part "or `None` if it hasn't been initialized" does not apply anymore). In fact, I feel with your changes, it would make more sense if we replaced the name of the `data` field with `animated_objects`, the `InitializedData` struct could be renamed to `AnimatedObject` and all the `init_data` references could be called `animated_object` instead.

I'm not set in on this naming scheme, feel free to adapt as you wish :) But I think something more along those lines would make things clearer.

Reply:
Yea this makes sense!